File: rsetdocs/src/main/java/com/example/rsetdocs/controller/TestController.java
package com.example.rsetdocs.controller;

import com.example.rsetdocs.dto.HelloRequest;
import com.example.rsetdocs.dto.HelloResponse;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;


@RestController
public class TestController {

    @GetMapping("/api/v1/hello")
    public ResponseEntity<HelloResponse> hello(HelloRequest helloRequest) {
        HelloResponse helloResponse = new HelloResponse();
        helloResponse.setId(helloRequest.getId());
        helloResponse.setReplyMessage("reply: " + helloRequest.getMessage());

        return ResponseEntity.ok(helloResponse);
    }
}

File: jdsl-example/src/main/kotlin/com/example/jdslexample/presentation/api/TestController.kt
package com.example.jdslexample.presentation.api

import com.example.jdslexample.persistence.repository.MemberQueryRepository
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class TestController(
    val memberQueryRepository: MemberQueryRepository
) {

    @GetMapping("/test")
    fun test(): ResponseEntity<String> {
        memberQueryRepository.queryUtilTest()
        return ResponseEntity.ok("1234")
    }
}